src/lib.rs

```diff
@@ -20,6 +20,8 @@ pub use self::parser::{ErrorKind, KeyMetadata, MetadumpResponse, StatsResponse,
 mod value_serializer;
 pub use self::value_serializer::AsMemcachedValue;
 
+const MAX_KEY_LENGTH: usize = 250;
```

Comment:
Might be worth linking to the Memcached documentation where this is defined

Reply:
Done.